pkg/rclone/nodeserver.go

```diff
@@ -235,3 +254,17 @@ func validateUnPublishVolumeRequest(req *csi.NodeUnpublishVolumeRequest) error {
 func (*nodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandVolumeRequest) (*csi.NodeExpandVolumeResponse, error) {
 	return nil, status.Errorf(codes.Unimplemented, "method NodeExpandVolume not implemented")
 }
```

Comment:
the concept of expanding a volume makes no sense for our mounts, so since we don't advertise having `ControllerServiceCapability_RPC_EXPAND_VOLUME` capability, calling this should error.